zerver/views/invite.py

```diff
@@ -187,7 +187,9 @@ def generate_multiuse_invite_backend(
     invite_expires_in_minutes: Optional[int] = REQ(
         json_validator=check_none_or(check_int), default=INVITATION_LINK_VALIDITY_MINUTES
     ),
-    invite_as: int = REQ(json_validator=check_int, default=PreregistrationUser.INVITE_AS["MEMBER"]),
+    invite_as: int = REQ(
+        json_validator=check_smallint(), default=PreregistrationUser.INVITE_AS["MEMBER"]
```

Comment:
Should this do `check_int_in(list(PreregistrationUser.INVITE_AS.values()))` instead?

(Similarly for `invite_users_backend`, which already does that check in the body of the function.)

Reply:
I think it should - that's a much more concise way to handle this, thanks!